src/letter_grade.js

```diff
@@ -0,0 +1,65 @@
+function HTMLStrLetterGrade()
+{
+    const studentIds = database.database.GetStudentIDs()
+    const numberOfStudents = studentIds.length
+
+    const studentData = studentIds.map((studentId) =>{
+        return {
+            id: studentId,
+            grade: database.database.GetLetterGrade(studentId)
+        }
+    })
+    console.log(studentData[0])
+    
+    return `
+    <div ><h2>SOEN 287 Section Q </h2> </div>
+    <div><h3>Assign letter grade for the students</h3></div>
+    <div><h4>ID &emsp;&emsp;&emsp;&emsp; Letter Grade</h4></div>
+
+    <style> 
+    input {
```

Comment:
the style here should not be in your html code. For now please remove it, the webpage will not look great but once Nikita has completed the style you will be able to reference the style he created in the css files